README.md

```diff
@@ -8,132 +8,92 @@
 
 An [anagram](https://en.wikipedia.org/wiki/Anagram) is a word or phrase formed by rearranging the letters of a different word or phrase. _Adagrams_ is a fictional game in which a player is given a random set of letters and must make an anagram using those letters. _Adagrams_ has a specific scoring system, so that the player's submitted anagram scores points. The rules for Adagrams are roughly inspired by the "Letter Round" portion of the British game show [Countdown](https://en.wikipedia.org/wiki/Countdown_%28game_show%29%23Letters_round).
 
-While working on _Adagrams_, it may help to think of a physical metaphor for this game, such as other common word games like _Scrabble_ or _Bananagrams_. These analog games all feature a _pile_ of letter _tiles_ that the player _draws_ from.
+While working on _Adagrams_, it may help to think of a physical metaphor for this game, such as other common word games like _Scrabble_ or _Bananagrams_. These analog games all feature a _pool_ of letter _tiles_ that the player _draws_ from.
+
+In this version of _Adagrams_, we will only be working with the English alphabet.
```

Comment:
It would probably be helpful to have formal definitions for these terms, something like this:

Term | Definition | Representation
---    | ---  | ---
Letter or Tile | A single English capital letter | 1-character string (`"A"`, `"B"`, etc)
Letter Pool | The set of 98 letters (9 `A`s, 2 `B`s, etc) used to play the game | Up to you (see wave 1)
Hand | A set of 10 tiles that the player can use to construct a word | An array of ten 1-character strings, like `["A", "P", "K", "A", "E", "F", "W", "H", "F", "Q"]`
Word | A word submitted by the user | `"FISH"`, `"DEVELOPERS"`, `"ZZHYT"`, `"A"`
Drawing | Constructing a new hand using letters from the pool

Reply:
Not sure if this is exactly the right place for it.

Reply:
Idea: We put this in a `hint.md` file, and we link to that in the Readme. This table, imo, can have a paragraph that talks about, "given a problem, as programmers, we think about it conceptually, and then try to transfer the problem's factors into representations of data. What data type is it?" and then give them this table